rkt/image_rm.go

```diff
@@ -39,34 +39,51 @@ func rmImages(s *store.Store, images []string) error {
 	errors := 0
 	staleErrors := 0
 	for _, pkey := range images {
+		var keys []string
 		errors++
 		h, err := types.NewHash(pkey)
 		if err != nil {
-			stderr("rkt: wrong image ID %q: %v", pkey, err)
-			continue
-		}
-		key, err := s.ResolveKey(h.String())
-		if err != nil {
-			stderr("rkt: image ID %q not valid: %v", pkey, err)
-			continue
-		}
-		if key == "" {
-			stderr("rkt: image ID %q doesn't exist", pkey)
-			continue
+			var found bool
+			keys, found, err = s.ResolveName(pkey)
+			if len(keys) > 0 {
+				errors += len(keys) - 1
+			}
+			if !found {
+				stderr("rkt: image name %q not found", pkey)
+				continue
+			}
+			if err != nil {
+				stderr("rkt: image name %q not valid: %v", pkey, err)
+				continue
+			}
+		} else {
+			key, err := s.ResolveKey(h.String())
+			if err != nil {
+				stderr("rkt: image ID %q not valid: %v", pkey, err)
+				continue
+			}
+			if key == "" {
+				stderr("rkt: image ID %q doesn't exist", pkey)
+				continue
+			}
+
+			keys = append(keys, key)
 		}
 
-		if err = s.RemoveACI(key); err != nil {
-			if serr, ok := err.(*store.StoreRemovalError); ok {
-				staleErrors++
-				stderr("rkt: some files cannot be removed for image ID %q: %v", pkey, serr)
-			} else {
-				stderr("rkt: error removing aci for image ID %q: %v", pkey, err)
-				continue
+		for _, key := range keys {
+			if err = s.RemoveACI(key); err != nil {
+				if serr, ok := err.(*store.StoreRemovalError); ok {
+					staleErrors++
+					stderr("rkt: some files cannot be removed for image %q: %v", pkey, serr)
+				} else {
+					stderr("rkt: error removing aci for image %q: %v", pkey, err)
+					continue
+				}
 			}
+			stdout("rkt: successfully removed aci for image: %q", pkey)
```

Comment:
If I understand correctly, if you have 3 etcd images, it will print:
```
# rkt image rm coreos.com/etcd
rkt: successfully removed aci for image: "coreos.com/etcd"
rkt: successfully removed aci for image: "coreos.com/etcd"
rkt: successfully removed aci for image: "coreos.com/etcd"
rkt: 3 image(s) successfully removed
```

Maybe it should print both the image name and the image id on each line?